cmd/kubeadm/app/constants/constants.go

```diff
@@ -191,17 +188,17 @@ const (
 	// MasterConfigurationConfigMapKey specifies in what ConfigMap key the master configuration should be stored
 	MasterConfigurationConfigMapKey = "MasterConfiguration"
 
-	// KubeletBaseConfigurationConfigMap specifies in what ConfigMap in the kube-system namespace the initial remote configuration of kubelet should be stored
-	KubeletBaseConfigurationConfigMap = "kubelet-base-config-1.9"
+	// KubeletBaseConfigurationConfigMapPrefix specifies in what ConfigMap in the kube-system namespace the initial remote configuration of kubelet should be stored
+	KubeletBaseConfigurationConfigMapPrefix = "kubelet-config-"
 
 	// KubeletBaseConfigurationConfigMapKey specifies in what ConfigMap key the initial remote configuration of kubelet should be stored
 	KubeletBaseConfigurationConfigMapKey = "kubelet"
 
-	// KubeletBaseConfigurationDir specifies the directory on the node where stores the initial remote configuration of kubelet
-	KubeletBaseConfigurationDir = "/var/lib/kubelet/config/init"
+	// KubeletBaseConfigMapRolePrefix defines the base kubelet configuration ConfigMap.
+	KubeletBaseConfigMapRolePrefix = "kubeadm:kubelet-config-"
 
-	// KubeletBaseConfigurationFile specifies the file name on the node which stores initial remote configuration of kubelet
-	KubeletBaseConfigurationFile = "kubelet"
+	// KubeletConfigurationFile specifies the file name on the node which stores initial remote configuration of kubelet
+	KubeletConfigurationFile = "/var/lib/kubelet/config.yaml"
```

Comment:
If I am understanding the docs for [Dynamic Kubelet Config](https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/), it looks like the dynamic config is downloaded from the config map to the directory specified by `--dynamic-config-dir` automatically and the config pulled down overrides the values that are set through `--config`. With that being the case, shouldn't we write the static config file (or initial config in the case of dynamic config) to `/etc/kubernetes` and leave `/var/lib/kubelet` for the content that is dynamically managed at runtime by the kubelet?